TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOp/LinearIntakeEndUnitRotatorTeleOp.java

```diff
@@ -0,0 +1,40 @@
+package org.firstinspires.ftc.teamcode.TeleOp;
+
+import com.arcrobotics.ftclib.command.button.Trigger;
+import com.arcrobotics.ftclib.gamepad.GamepadKeys;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.teamcode.Libraries.MMLib.MMOpMode;
+import org.firstinspires.ftc.teamcode.MMRobot;
+import org.firstinspires.ftc.teamcode.utils.OpModeType;
+
+
+@TeleOp
+public class LinearIntakeEndUnitRotatorTeleOp extends MMOpMode {
+    MMRobot robotInstance = MMRobot.getInstance();
+
+    public LinearIntakeEndUnitRotatorTeleOp(){
+        super(OpModeType.NonCompetition.EXPERIMENTING);
+    }
+
+    @Override
+    public void onInit() {
+
+        robotInstance.mmSystems.initRobotSystems();
+        Trigger leftTriggerCondition = new Trigger(
+                () -> robotInstance.mmSystems.gamepadEx2.getTrigger(GamepadKeys.Trigger.LEFT_TRIGGER) > 0.05
+        );
+        leftTriggerCondition.whenActive(
+                robotInstance.mmSystems.linearIntake.setPositionByJoystick(()-> gamepad2.left_trigger));
+
+        waitForStart();
```

Comment:
remove `waitForStart();` there is no need for it it was my mistake telling you to add it